src/components/header/header.js

```diff
@@ -1,26 +1,64 @@
 import React, {Component} from 'react';
-import {MainMenu} from '../../components';
-
+import {MainMenu, ModalAuth} from '../../components';
 /**
- * Header of project
- */
+* Header of project
+*/
 class Header extends Component {
   /**
- * render Header
- * @return {html} The template of Header class
- */
-  render() {
-    return (
-      <header className="header">
-        <h1 className="header__logo">MASTERCLASS VIET NAM</h1>
+  * render Header
+  * @param {any} props
+  */
+  constructor(props) {
+    super(props);
+    this.handleModalLogin = this.handleModalLogin.bind(this);
+  }
+  /**
+  * render Header
+  * @param {any} e
+  */
+  handleModalLogin(e) {
+    e.preventDefault();
+    this.props.initAction.showModal('modalAuth');
+  }
+  /**
+  * render Header
+  * @return {any}
+  */
+  checkLogin() {
+    if (this.props.payload.userReducer.loginStatus === true) {
+      return (
         <ul className="header__nav-language">
+          <li><a href="">Vietnamese</a></li>
           <li>
-            <a href="">Vietnamese</a>
+            <a>
+              xin chào {this.props.payload.userReducer.userInfo.userName}
+            </a>
           </li>
+          <li><a href="#" onClick={()=>this.props.userAction.signOut()}>Đăng xuất</a></li>
```

Comment:
@ndduy already implemented in  [https://github.com/dinosysOrg/masterclass-web/pull/24/commits/7f151e4326acfbe08beaecf92a7b8f0e14da759d](url)